.github/workflows/build_and_functional_tests.yml

```diff
@@ -0,0 +1,78 @@
+name: Build and run functional tests using ragger through reusable workflow
+
+# This workflow will Lint, Build and then Run tests.
+
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      - main
+      - develop

```

Comment:
We don't have a develop branch :man_shrugging:

Reply:
Fixed